site/components/case-study-overview/index.js

```diff
@@ -1,29 +1,27 @@
 import React from 'react';
 import InlineSVG from 'svg-inline-react';
+import moment from 'moment';
 import styles from './style.css';
 
-/* PNG logo imports */
-import bbcPNG from './PNG/bbc.png';
-import bmwPNG from './PNG/bmw.png';
-import camdenPNG from './PNG/camden.png';
-import cartrawlerPNG from './PNG/cartrawler.png';
-import financialTimesPNG from './PNG/financialtimes.png';
 import fortnumPNG from './PNG/fortnum.png';
-import hsbcPNG from './PNG/hsbc.png';
-import selfridgesPNG from './PNG/selfridges.png';
-import skyPNG from './PNG/sky.png';
-import tescoPNG from './PNG/tesco.png';
+import financialTimesPNG from './PNG/financialtimes.png';
+import arrowSVG from '../../../assets/images/SVG/arrow.svg';
+import ftFigureSVG from './SVG/ft-figure.svg';
+
 import Link from '../link';
+import CaseStudyCompanyLogos from '../case-study-company-logos';
 
-/* SVG figure imports */
-import ftFigureSVG from './SVG/ft-figure.svg';
-import fortnumFigureSVG from './SVG/fortnum-figure.svg';
-import camdenFigureSVG from './SVG/camden-figure.svg';
-import skyFigureSVG from './SVG/sky-figure.svg';
-import arrowSVG from '../../../assets/images/SVG/arrow.svg';
+const daysSinceFortumStarted = () => {
+  const startDate = moment([2014, 5, 14]);
```

Comment:
Start date should be [2014, 4, 14]. The month count starts at 0, so May will be 4.